File: 2137-final-value-of-variable-after-performing-operations/2137-final-value-of-variable-after-performing-operations.c
int finalValueAfterOperations(char** operations, int operationsSize) {
    int result=0;
    for(int i=0; i<operationsSize; i++){
        if((operations[i][0] == '+' && operations[i][1] == '+')||(operations[i][1] == '+' && operations[i][2] == '+')){
            result+=1;
        } else if((operations[i][0] == '-' && operations[i][1] == '-')||(operations[i][1] == '-' && operations[i][2] == '-')){
            result-=1;
        }
    }
    return result;
}

File: 2137-final-value-of-variable-after-performing-operations/2137-final-value-of-variable-after-performing-operations.java
class Solution {
    public int finalValueAfterOperations(String[] operations) {
        int x = 0;
        for(int i=0; i<operations.length; i++){
            String operator = operations[i];
            if(operator.equals("++X") || operator.equals("X++")) x++;
            else x--;
        }
        return x;
    }
}